conda_smithy/configure_feedstock.py

```diff
@@ -102,10 +105,10 @@ def render_circle(jinja_env, forge_config, forge_dir):
 
 @contextmanager
 def fudge_subdir(subdir):
-    orig = conda_build.metadata.cc.subdir
-    conda_build.metadata.cc.subdir = subdir
+    orig = cc.subdir
+    cc.subdir = subdir
     yield
-    conda_build.metadata.cc.subdir = orig
+    cc.subdir = orig
```

Comment:
Ugh, or I could just do it right.  I had it this way, but then lost it in reversion of some mock stuff that didn't end up working.